integration-tests/tests/aws/kbv-check-happy.test.ts

```diff
@@ -0,0 +1,67 @@
+import { stackOutputs } from "./resources/cloudformation-helper";
+import { clearItems, populateTable } from "./resources/dynamodb-helper";
+import { executeStepFunction } from "./resources/stepfunction-helper";
+
+describe("HMRC KBV Check ", () => {
+  const stateMachineInput = {
+    sessionId: "12345",
+  };
+  const testUser = {
+    sessionId: "12345",
+    nino: "AA000003D",
+  };
+
+  const testQuestions = [
+    {
+      sessionId: "12345",
+      questionKey: "ita-bankaccount",
+      answered: "false",
+      correlationId: "93dcc67c-fe6d-4bd7-b68f-2bd848e0d572",
+      info: new Map(),
+    },
+    {
+      sessionId: "12345",
+      questionKey: "ita-bankaccount2",
+      answered: "false",
+      correlationId: "93dcc67c-fe6d-4bd7-b68f-2bd848e0d573",
+      info: new Map(),
+    },
+  ];
+
+  let output: Partial<{
+    PersonalIdenityTable: string;
+    QuestionsTable: string;
+    QuestionStateMachineArn: string;
+  }>;
+  beforeEach(async () => {
+    output = await stackOutputs(process.env.STACK_NAME);
+    await populateTable(testUser, output.PersonalIdenityTable);
+    testQuestions.forEach(
+      async (question) => await populateTable(question, output.QuestionsTable)
+    );
+  });
+
+  afterEach(async () => {
+    await clearItems(output.PersonalIdenityTable as string, {
+      sessionId: testUser.sessionId,
+    });
+    testQuestions.forEach(
+      async (question) =>
+        await clearItems(output.QuestionsTable as string, {
+          sessionId: question.sessionId,
+          questionKey: question.questionKey,
+        })
+    );
+  });
+  describe("Happy Case GetQuestion Check", () => {
```

Comment:
Can we make all the `describe` and `it` blocks more meaningful to what exactly is being tested and so it's obvious to the reader what the current state and context of the test is?